docs/get-started/cost-billing/how-pricing-works.md

```diff
@@ -32,9 +32,9 @@ and billing model.
 
 | Name         | vGPU | vCPU | RAM (GiB) | Current Price |
 | ------------ | ---- | ---- | --------- | ------------- |
-| H100 GPU     | 1    | 63   | 376       | $6.04         |
-| A100sxm4 GPU | 1    | 31   | 240       | $2.078        |
-| A100 GPU     | 1    | 24   | 74        | $1.803        |
+| H100 GPU     | {{su_info_dict["GPUH100"]["vGPUs"]}}    | {{su_info_dict["GPUH100"]["vCPUs"]}}   | {{su_info_dict["GPUH100"]["RAM"]}}       | ${{su_info_dict["GPUH100"]["rate"]}}        |
```

Comment:
@joachimweyl @knikolla I was wondering, is it a good idea to turn this into a for loop that goes over all Non-BM SUs? Just to remove some repetition

Reply:
If that is easy enough to code and it is less code, that sounds good to me, but I will defer to @knikolla on coding best practices.

Reply:
@QuanMPhm : did you test this locally by running `mkdocs serve` ?